integr8ly/installation
# INTLY-10338 - delete zync ServiceMonitor

## Additional Information

JIRA: https://issues.redhat.com/browse/INTLY-10338
We decided that it would be best to removing this ServiceMonitor, since it's hasn't been working since 2.x GA and zync metrics are not available in RHMI 2.x and RHOAM.

## Verification Steps
As the verifier of the PR the following process should be done:
Run upgrade playbook and verify that "3scale-zync-servicemonitor" ServiceMontior was removed from 3scale ns
OR carefully review following log:
https://gist.github.com/matskiv/b41ae842f33023b5e302c3210d6094c6#file-upgrade-log-intly-10338

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 



## Checklist



Is an upgrade task required and are there additional steps needed to test this?
